src/runtime_src/core/pcie/driver/linux/xocl/subdev/ps.c

```diff
@@ -0,0 +1,202 @@
+/*
+ * A GEM style device manager for PCIe based OpenCL accelerators.
```

Comment:
overall, yes, but doesn't apply to ps.c

Reply:
I got this from microblaze.c. Anyway, I have updated ps.c